Makefile

```diff
@@ -10,19 +10,46 @@ GOHOSTOS=$(shell go env GOHOSTOS)
 GOHOSTARCH=$(shell go env GOHOSTARCH)
 GO_MOD_VERSION=$(shell grep "^go" go.mod | awk '{print $$2}')
 GO_INSTALLED_VERSION=$(shell go version | awk '{print $$3}' | sed -e /.*go/s///)
-export CGO_ENABLED=0
 
+# Build number passed in must be a monotonic int representing
+# the build.
+JUJU_BUILD_NUMBER ?=
+
+# JUJU_VERSION is the JUJU version currently being represented in this
+# repository.
```

Comment:
Can we avoid splitting the comment just to have "repository" on its own line?

Reply:
Trying to keep it wrapped under 80 chars